src/core.py

```diff
@@ -0,0 +1,14 @@
+import os
+import jinja2
+
+JINJA_ENVIRONMENT = jinja2.Environment(
+    loader=jinja2.FileSystemLoader(os.path.dirname(__file__) + "/../static/template"),
```

Comment:
Maybe add a comment here. I had a really difficult time figuring out how this worked the first time I tried to use this